### DISCUSSION TITLE
rocketmq ipv6改动

### DISCUSSION THREAD

**Discussion by GitHubMiracle** [apache/rocketmq#8690] [Q&A]
rocketmq支持ipv6吗，请问有没有案例可以参考下，5.1.3做ipv6改动的时候namesrv连接broker失败。
Does RocketMQ support IPv6? Are there any cases that I can refer to? When making IPv6 modifications in version 5.1.3, the connection between namesrv and broker fails.

**Comment by jaylu163** [comment]
支持ipv6，可以搭建玩起来
